src/qt/rpcconsole.cpp

```diff
@@ -1018,11 +1018,9 @@ void RPCConsole::updateTrafficStats(quint64 totalBytesIn, quint64 totalBytesOut)
 
 void RPCConsole::peerLayoutAboutToChange()
 {
-    QModelIndexList selected = ui->peerWidget->selectionModel()->selectedIndexes();
     cachedNodeids.clear();
-    for(int i = 0; i < selected.size(); i++)
-    {
-        const CNodeCombinedStats *stats = clientModel->getPeerTableModel()->getNodeStats(selected.at(i).row());
+    for (const auto& peer : GUIUtil::getEntryData(ui->peerWidget, PeerTableModel::NetNodeId)) {
+        const auto stats = peer.data(PeerTableModel::StatsRole).value<CNodeCombinedStats*>();
```

Comment:
6e27ee8b6 To me at least, this would be easier to understand what is going on (and not using `auto` allows uniform initialization). The same motivation for the other suggestions below.
```diff
-    for (const auto& peer : GUIUtil::getEntryData(ui->peerWidget, PeerTableModel::NetNodeId)) {
-        const auto stats = peer.data(PeerTableModel::StatsRole).value<CNodeCombinedStats*>();
+    for (const QModelIndex& peer : GUIUtil::getEntryData(ui->peerWidget, PeerTableModel::NetNodeId)) {
+        const CNodeCombinedStats* stats{peer.data(PeerTableModel::StatsRole).value<CNodeCombinedStats*>()};
```

Reply:
[AAA Style (Almost Always Auto)](https://herbsutter.com/2013/08/12/gotw-94-solution-aaa-style-almost-always-auto/)

> ... easier to understand what is going on

Maybe `GUIUtil::getEntryData` requires renaming? `GetModelIndexesByColumn`?

@promag What do you think?

Reply:
Side node, I'd just ditch `getEntryData` and `hasEntryData`, doesn't improve that much over Qt API. If we keep them then maybe reword to `GetSelectedModelIndex` or something like that with "selection".

In this particular case I think `peer` should have the type, and leave auto for `status`.

Reply:
Re AAA Style, yes, Sutter states the caveat about readability is that people use IDEs. Here, where review is the primary constraint, whatever can aid reviewers in reading and reviewing the code seems like a win, so in practice I've found myself using `auto` only when the type is obvious or when it can avoid a complicated, verbose type. Anyway, just a minor suggestion, feel free to ignore.

Reply:
Thanks! [Updated](https://github.com/bitcoin-core/gui/pull/161#issuecomment-752735925).